README.md

```diff
@@ -1,25 +1,28 @@
+<img src="https://www.lsstcorporation.org/lincc/sites/default/files/PastedGraphic-8.png" width="300" height="100">
```

Comment:
Oo, the logo is a nice touch :)

Reply:
All credit to @dougbrn - he added it to the lsstseries readme